warplib/maths/bytes_access.cairo

```diff
@@ -0,0 +1,83 @@
+from warplib.maths.pow2 import pow2
+from starkware.cairo.common.bitwise import bitwise_and
+from warplib.maths.utils import narrow_safe
+
+from starkware.cairo.common.cairo_builtins import BitwiseBuiltin
+from starkware.cairo.common.uint256 import Uint256
+from starkware.cairo.common.math_cmp import is_le
+
+from starkware.cairo.common.serialize import serialize_word
+
+func byte_accessor{range_check_ptr}(pos : felt) -> (offset : felt):
```

Comment:
This seems like a very inefficient way to generate a mask. Worth checking the step counts, but I would have thought it'd be more efficient to either do `255 * pow2(index*8)` or `pow2((index+1)*8))-pow2(index*8)`, or even set up a function like pow2 for masks (in which case make sure to write the mask values in hexadecimal to keep things simple)